endpoints.json

```diff
@@ -71,6 +71,22 @@
         }]
     }
 
+  },
+  "PATCH /api/articles/:article_id/comments": {
```

Comment:
is this endpoint supposed to patch an article? if so the endpoint given is to comments?